src/index.ts

```diff
@@ -1,112 +1,205 @@
+// This shouldn't even be necessary, docs say props of objects shouldn't trigger this rule but it does?
+/* eslint-disable no-param-reassign */
 import { network, whisper } from '@oliveai/ldk';
 
-import { Event, EventCategory, EventName } from './types';
-
-export * from './types';
+import { Event, EventCategory, EventName, SegmentRequest } from './types';
 
 interface Config {
+  /** Identify the entire loop by name */
   loopName: string;
-  userId: string;
+  /** Write key provided by the Segment HTTP destination */
   writeKey: string;
+  /** Unique ID for the user, usually the sub prop from LDK User.jwt() */
+  userId: string;
+  /** Email for the user, will be stripped to only track the domain */
+  userEmail: string;
 }
 
 export default class Segment {
   static BASE_URL = 'https://api.segment.io/v1';
```

Comment:
Probably over kill so feel free to ignore should api version be a configurable option and added when building the url. Just a small thing to future proof the library. We could set the default to `v1` but have it configurable.

Reply:
Most likely, yes. That's probably going to be part of the configurable transports thing idea